components/function-runtimes/java11/src/main/java/io/project/kyma/serverless/handler/Handler.java

```diff
@@ -0,0 +1,16 @@
+package io.project.kyma.serverless.handler;
+
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+
+import io.project.kyma.serverless.sdk.CloudEvent;
+import io.project.kyma.serverless.sdk.Function;
+
+
+public class Handler implements Function {
+
+    @Override
+    public Response call(CloudEvent event, Context context) {
+        throw new IllegalStateException("Not implemented stub Handler");
```

Comment:
I decided to override this `Handler` class in function build, because I couldn't find a better process to compile the java function with graalvm and openjdk.